docs/content/toc.md

```diff
@@ -108,7 +110,8 @@ layout: toc
     * [Historical](/docs/VERSION/design/historical.html)
     * [MiddleManager](/docs/VERSION/design/middlemanager.html)
       * [Peons](/docs/VERSION/design/peons.html)
-    * [Realtime (Deprecated)](/docs/VERSION/design/realtime.html)
+    * [Router](/docs/VERSION/development/router.html) (optional; experimental)
+    * [Realtime Node](/docs/VERSION/design/realtime.html) (deprecated)
```

Comment:
why not just remove this entirely? I don't think deprecated nodes should be in the ToC

Reply:
Sure, removed it. Please review again.